src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorInitializerSymbolReferenceFinder.cs

```diff
@@ -49,8 +51,9 @@ protected override bool CanFind(IMethodSymbol symbol)
             }, cancellationToken);
         }
 
-        protected override async ValueTask<ImmutableArray<FinderLocation>> FindReferencesInDocumentAsync(
+        protected sealed override async ValueTask<ImmutableArray<FinderLocation>> FindReferencesInDocumentAsync(
             IMethodSymbol methodSymbol,
+            HashSet<string>? globalAliases,
             Document document,
             SemanticModel semanticModel,
             FindReferencesSearchOptions options,
```

Comment:
most finders were unaffected (except fort hese signature changes).  the only finders that care are Namespaces/NamedTypes/Constructors